scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -167,6 +180,26 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
         }
     }
 
+    private fun getResultsFile(scannerDetails: ScannerDetails, pkg: Package, outputDirectory: File): File {
```

Comment:
If `readFromStorage()` gets docs (see below), please add some here, too, even if the function is private, for consistency.

Reply:
Done.